clients/vault/resources/config/mainnet/stellar_relay_config_frankfurt.json

```diff
@@ -4,18 +4,13 @@
     "port": 11625
   },
   "node_info": {
-    "ledger_version": 21,
-    "overlay_version": 34,
-    "overlay_min_version": 32,
-    "version_str": "stellar-core 21.2.0 (d78f48eacabb51753e34443de7618b956e61c59f)",
+    "ledger_version": 22,
+    "overlay_version": 35,
+    "overlay_min_version": 33,
+    "version_str": "stellar-core 22.0.0 (721fd0a654d5e82d38c748a91053e530a475193d)",
     "is_pub_net": true
   },
   "stellar_history_archive_urls": [
-    "http://history.stellar.org/prd/core-live/core_live_001",
-    "http://history.stellar.org/prd/core-live/core_live_002",
-    "http://history.stellar.org/prd/core-live/core_live_003",
-    "https://stellar-history-de-fra.satoshipay.io",
-    "https://stellar-history-sg-sin.satoshipay.io",
-    "https://stellar-history-us-iowa.satoshipay.io"
+    "https://hercules-history.publicnode.org"
```

Comment:
Did you change this due to the 'overloaded' issue or for a different reason? If so, you'd rather need to change the `address` field in `connection_info` to the `host` you can also find in Stellarbeat. If not, I'll leave this comment as a reminder that we should change it back eventually.

Reply:
Back to original configs now.